### PR TITLE
Add 'unmanaged' to ol.layer.LayerState

### PR DESCRIPTION
**Pull request by ahocevar**

### CODE REVIEW

File: src/ol/layer/layerbase.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ ol.layer.LayerProperty = {
  *            saturation: number,
  *            sourceState: ol.source.State,
  *            visible: boolean,
+ *            unmanaged: (boolean|undefined),
```
Comment (elemoine) [inline src/ol/layer/layerbase.js]: We could also decide to always/systematically define it, but it's ok like this as well.

**Comment by elemoine** [comment]
I added a comment, but please merge with or without taking this comment into account.

**Comment by ahocevar** [comment]
@elemoine I changed the commit. Is this what you had in mind?

**Comment by elemoine** [comment]
Yes!

**Comment by marcjansen** [comment]
Would 'managed' be a better name? I've had bad experience most of the time when naming stuff with the negated variant

**Comment by ahocevar** [comment]
I like the idea @marcjansen  - if you're willing to create a PR :smile:

**Comment by marcjansen** [comment]
My bad, I had better kept my mouth shut :wink: I MAY have time tomorrow, for a PR

**Comment by ahocevar** [comment]
@marcjansen How about I create a PR and you review - ETA 10 min.

**Comment by marcjansen** [comment]
Ok, promised

**Comment by ahocevar** [comment]
@marcjansen #3793. Thanks.

**Comment by tschaub** [comment]
It would be great to have a bit of description here.  The diff clearly does what the title says, but some additional motivation or context would be useful for those reading a changelog.  Thanks!

**Comment by tschaub** [comment]
I see https://github.com/openlayers/ol3/pull/3758#issuecomment-111031177 now.  Just catching up on some significant changes!  Still, it would be nice to avoid descriptionless tickets.

**Pull request merged**